script.js

```diff
@@ -25,3 +26,186 @@ const Contact = document.querySelector('#Contact');
+    Technology: ['HTML', 'CSS', 'JavaScript'],
+    linkToLive: 'https://zafron047.github.io/Portfolio_Cohort-20_Mod-1/',
+    linkToSource: 'https://github.com/Zafron047/Portfolio_Cohort-20_Mod-1',
+  },
+  {
+    projectName: 'Tonic',
+    projectDescription:
+      'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    Technology: ['HTML', 'CSS', 'JavaScript'],
+    linkToLive: 'https://zafron047.github.io/Portfolio_Cohort-20_Mod-1/',
+    linkToSource: 'https://github.com/Zafron047/Portfolio_Cohort-20_Mod-1',
+  },
+  {
+    projectName: 'Tonic',
+    projectDescription:
+      'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    Technology: ['HTML', 'CSS', 'JavaScript'],
+    linkToLive: 'https://zafron047.github.io/Portfolio_Cohort-20_Mod-1/',
+    linkToSource: 'https://github.com/Zafron047/Portfolio_Cohort-20_Mod-1',
+  },
+  {
+    projectName: 'Tonic',
+    projectDescription:
+      'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    Technology: ['HTML', 'CSS', 'JavaScript'],
+    linkToLive: 'https://zafron047.github.io/Portfolio_Cohort-20_Mod-1/',
+    linkToSource: 'https://github.com/Zafron047/Portfolio_Cohort-20_Mod-1',
+  },
+];
+
+const projectBtns = document.querySelectorAll('.see-project');
+Array.from(projectBtns).forEach((projectBtns, i) => {
+  projectBtns.addEventListener('click', () => {
+    const viewportWidth = window.innerWidth || document.documentElement.clientWidth;
+
+    if (viewportWidth >= 768) {
+      const popDiv = document.createElement('div');
+      popDiv.id = 'blur-background';
+      popDiv.classList.add('blur-background');
+      popDiv.style.position = 'fixed';
+      popDiv.style.display = 'block';
+      popDiv.style.top = '0';
+      popDiv.style.left = '0';
+      popDiv.style.width = '100vw';
+      popDiv.style.height = '100vh';
+      popDiv.style.zIndex = '11';
+      popDiv.style.backdropFilter = 'blur(6px)';
+      popDiv.innerHTML = `
+        <div class='d-card'>
+          <div class='D-Pop-Header'>
```

Comment:
- [ ] Your mobile pop up is somehow fixed to the bottom layer with just the upper most part showin with a cancel button. You did a great job on the desktop pop up, of which i suggest to consider simialar implementaion to have the mobile one center on the screen just as on the figma design.

## Your view:
![image](https://github.com/Zafron047/Portfolio_Cohort-20_Mod-1/assets/50990925/f607ef8d-f681-4390-b05b-01caa4c86300)

## Figma:
![Uploading image.png…]()

Reply:
@Shedrack-Sunday I have been able to fix the position of mobile popup however, all the content with it's original size units given by figma cannot be stored within this popup without `Y Overflow`. So I had to give the `overflow auto` for this popup otherwise the whole popup including the buttons don't show cause the height is not enough.